ansible/roles/jenkins-node/tasks/main.yaml

```diff
@@ -0,0 +1,60 @@
+- name: Create jenkins group.
+  group:
+    name: jenkins
+
+- name: Create jenkins user.
+  user:
+    createhome: yes
+    group: jenkins
+    home: /home/jenkins
+    name: jenkins
+
+- name: Make .ssh dir for jenkins
+  file:
+    path: /home/jenkins/.ssh
+    state: directory
+    mode: 0700
+    owner: jenkins
+    group: jenkins
+
+- name: Install ssh key for jenkins
+  authorized_key:
```

Comment:
The authorized _key module already handles the creation of the .ssh directory, from the docs:

```
Prameter manage_dir:

Whether this module should manage the directory of the authorized key file. If set, the module will create the directory, as well as set the owner and permissions of an existing directory. Be sure to set manage_dir=no if you are using an alternate directory for authorized_keys, as set with path, since you could lock yourself out of SSH access.
```